transport/src/main/java/io/netty/bootstrap/Bootstrap.java

```diff
@@ -297,6 +297,20 @@ public Bootstrap clone(EventLoopGroup group) {
         return bs;
     }
 
+    /**
+     * Return the configured remote address or {@code null} if non is configured yet.
+     */
+    public SocketAddress remoteAddress() {
+        return remoteAddress;
+    }
+
+    /**
+     * Return the configured {@link AddressResolverGroup} or the default if non is configured yet.
+     */
+    public AddressResolverGroup<?> resolver() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1452)